src/main/java/core/basesyntax/serviceimpl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.storage.Storage;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    public static final String FIRST_LINE = "fruit,quantity";
```

Comment:
The constant `FIRST_LINE` should be private if it is not used outside of this class. This is part of following the encapsulation principle.